src/arbitraries/util.ts

```diff
@@ -13,6 +13,14 @@ export function stringify(object: any) {
   return (object instanceof Object || object instanceof Array) ? JSON.stringify(object) : object
 }
 
+//Splitmix64 code adapted from https://prng.di.unimi.it/splitmix64.c
```

Comment:
Why are we not using a library implementing splitmix64? For example https://www.npmjs.com/package/@aureooms/js-pseudo-random

The code is always the same, but still...
https://aureooms.github.io/js-pseudo-random/file/src/splitmix64.js.html

This is an honest question, do we want a new dependency because of 5 lines of code?

Reply:
Also, why are we using splitmix64? I only talked about this particular RNG after looking at some nice pictures. You should investigate others...

Reply:
Actually, you didn't even mentioned splitmix64, but xoshiro128.

Reply:
The creator of xoshiro mentions the use of splitmix64 for the creation of the seed, but I believe that is because Splitmix64 is the default pseudo-random number generator algorithm in Java and the code on the creator's website is in java, so I removed the use of this particular algorithm. As to the choice of xoshiro, after browsing a bit, according to a comment on [this thread](https://stackoverflow.com/questions/521295/seeding-the-random-number-generator-in-javascript), It is the fastest generator that offers a 128-bit state.